src/settingsmanager.h

```diff
@@ -22,23 +22,26 @@ class SettingsManager : public QObject
     bool settingsExists(const QString &key);
     QVariant getSettings(const QString &key);
     qreal getZoomFactorByZimId(const QString &id);
+    int getKiwixServerPort() const { return m_kiwixServerPort; }
+    QString getKiwixServerIpAddress() const { return m_kiwixServerIpAddress; }
+    qreal getZoomFactor() const { return m_zoomFactor; }
+    QString getDownloadDir() const { return m_downloadDir; }
+    QString getMonitorDir() const { return m_monitorDir; }
 
 public slots:
     void setKiwixServerPort(int port);
-    int getKiwixServerPort() { return m_kiwixServerPort; };
     void setKiwixServerIpAddress(QString ipAddress);
-    QString getKiwixServerIpAddress() { return m_kiwixServerIpAddress; };
     void setZoomFactor(qreal zoomFactor);
-    qreal getZoomFactor() { return m_zoomFactor; };
     bool setDownloadDir(QString downloadDir);
-    QString getDownloadDir() { return m_downloadDir; }
+    bool setMonitorDir(QString monitorDir);
```

Comment:
Return type `bool` (as copy-pasted from `setDownloadDir()`) is meaningless with the current implementation. Change it to `void` (and fix the return type of ``setDownloadDir()` too, but in a separate non-fixup commit).

Reply:
> this commit should be before "Add monitor directory entry" right?

Yes

> Also, can I rebase the current 3 fixups?

Please don't rebase now. Let me see the last two fixes and then you will rebase the entire PR right before it is merged.